06-building-a-program-start-to-finish/solving-the-problem/output/invertedIndex.js

```diff
@@ -0,0 +1,221 @@
+/* global FileReader */
+/* global InvertedIndex */
+/**
+ * @class InvertedIndex
+ * @classdesc blah blah
+ */
+/* eslint-disable */
+class InvertedIndex {
+  /* eslint-enable */
+  /**
+   * * @constructor
+   * initialises the class base properties
+   */
+  constructor() {
+    this.indicies = {};
+    this.indexedFiles = {};
+    this.uploadedFiles = {};
+  }
+  /**
+   * @createIndex method
+   * @param {fileName} fileName
+   * @param {fileContent} fileContent
+   * @returns {boolean}
+   * create index of the fileName
+   */
+  createIndex(fileName, fileContent) {
+    this.indicies[fileName] = this.indicies[fileName] || {};
+    const numOfBooks = fileContent.length;
```

Comment:
Please be descriptive with your naming. Abbreviations are not allowed. `numOfBooks`